### PR TITLE
Removes _themes/slides/*

### PR DESCRIPTION
**Pull request by kagarmoe**
Signed-off-by: kagarmoe <kgarmoe@chef.io>

### Description

We have an old _themes/slides directory that contains outdated information and does not appear related to our current theme. It causes problems for us in our archiving script, so I'm removing it.

### Definition of Done

Confirmation that this directory really doesn't apply to our current theme.

### Check List

- [x] Spell Check
- [x] Local build
- [x] Examine the local build
- [ ] All tests pass

### CODE REVIEW

**Comment by mjingle** [comment]
Local build looks okay to me. Will wait on @magwalk's approval before final approval & merging.

File: _themes/slides/static/js/jquery.min.js
Comment (magwalk) [inline _themes/slides/static/js/jquery.min.js:1]: My only question/concern is about removing this file. I am trying to get this running locally but my machine can't seem to get it going. So does everything seem to function correctly even after removing this file? If so then I'm all 👍

**Comment by kagarmoe** [comment]
@mjingle and I can both build the docs from this branch.

**Review by mjingle — APPROVED** [review]
tenor-222337923

**Pull request merged**
